CMakeLists.txt

```diff
@@ -21,6 +21,7 @@ if(NOT COMPILER_SUPPORTS_CXX11)
   message(FATAL_ERROR "HighFive version >= 2.0 requires c++ standard >= c++11")
 endif()
 
+
```

Comment:
Is it on purpose?

Reply:
not really. addressing your review I will take care of minimizing changes